### PR TITLE
Fix the broken links

### PR DESCRIPTION
**Pull request by arpanetus**
Change hyphen to underscore
Fix typo "perfix" into "prefix"

### CODE REVIEW

File: docs/piscine-rust-curriculum.md [hunk 1]
```diff
@@ -10,14 +10,14 @@ Exercices:
 
 - fibonacci2 | https://github.com/01-edu/public/tree/master/subjects/fibonacci2
 - scalar | https://github.com/01-edu/public/tree/master/subjects/scalar
-- temperature-conv | https://github.com/01-edu/public/tree/master/subjects/temperature-conv
+- temperature-conv | https://github.com/01-edu/public/tree/master/subjects/temperature_conv
```
Comment (MSilva95) [inline docs/piscine-rust-curriculum.md:13]: As this is your branch can you please do the same thing to the begging of the line for all of them?

```
- temperature_conv | https://github.com/01-edu/public/tree/master/subjects/temperature_conv
```

File: docs/piscine-rust-curriculum.md [hunk 2]
```diff
@@ -10,14 +10,14 @@ Exercices:
```
Comment (MSilva95) [inline docs/piscine-rust-curriculum.md:10]: As this is your branch can you please Change 
"Exercices:" to "Exercises:"

**Review by MSilva95 — CHANGES_REQUESTED** [review]

File: docs/piscine-rust-curriculum.md (hunk 1 above)
Comment (arpanetus) [reply docs/piscine-rust-curriculum.md:13]: oh, ok

File: docs/piscine-rust-curriculum.md (hunk 2 above)
Comment (arpanetus) [reply docs/piscine-rust-curriculum.md:10]: kk

File: docs/piscine-rust-curriculum.md (hunk 1 above)
Comment (arpanetus) [reply docs/piscine-rust-curriculum.md]: @MSilva95 done

File: docs/piscine-rust-curriculum.md (hunk 2 above)
Comment (arpanetus) [reply docs/piscine-rust-curriculum.md:10]: @MSilva95 done

**Comment by arpanetus** [comment]
@Frenchris @MSilva95 👀

**Pull request merged**